packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -317,6 +307,22 @@ export class FauxClassGenerator {
     return this.successExit();
   }
 
+  private async fetchObjects(
+    describe: SObjectDescribe,
+    types: string[]
+  ): Promise<SObject[]> {
+    const batchSize = 25;
```

Comment:
Let's add a note here that explains that `25` is the limit of sobjects supported by the describe api.

Reply:
@lcampos I added a constant instead. I thought that would be clearer.

Reply:
@rcoringrato-sfdc that works too 👍